### PR TITLE
[14.0] [MIG] sale_product_set_sale_by_packaging: Migrate to version 14.0

### PR DESCRIPTION
**Pull request by sonhd91**

### CODE REVIEW

File: sale_product_set_sale_by_packaging/__manifest__.py [hunk 1]
```diff
@@ -15,9 +15,5 @@
     "installable": True,
     "auto_install": True,
     "depends": ["sale_by_packaging", "sale_product_set_packaging_qty"],
-    "data": [
-        "views/product_set.xml",
```
Comment (simahawk) [inline sale_product_set_sale_by_packaging/__manifest__.py]: why `product_set.xml` removed?

**Review by simahawk — CHANGES_REQUESTED** [review]
can you please include here my last PR https://github.com/OCA/sale-workflow/pull/1735 ?

File: sale_product_set_sale_by_packaging/__manifest__.py (hunk 1 above)
Comment (sonhd91) [reply sale_product_set_sale_by_packaging/__manifest__.py]: I changed it to` product_set_line.xml` more suitable name with model `product.set.line`

**Review by simahawk — APPROVED** [review]

File: sale_product_set_sale_by_packaging/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Camptocamp SA
+# @author Simone Orsi <simone.orsi@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Sale Product Set Sale by packaging",
+    "summary": """
+    Glue module between `sale_by_packaging` and `sale_product_set_packaging_qty`.
+    """,
+    "version": "14.0.1.0.0",
+    "development_status": "Alpha",
```
Comment (rousseldenis) [inline sale_product_set_sale_by_packaging/__manifest__.py:10]: @sonhd91 @simahawk Still Alpha as second version ?

**Review by rousseldenis — APPROVED** [review]
Code review

File: sale_product_set_sale_by_packaging/__manifest__.py (hunk 2 above)
Comment (simahawk) [reply sale_product_set_sale_by_packaging/__manifest__.py:10]: Well, technically this is not the 2nd version, is the same one as on 13.0 and - so far - not tested live as the one on 13.0 indeed.

**Pull request merged**